test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/KeepAliveTimeoutTests.cs

```diff
@@ -50,17 +49,7 @@ public class KeepAliveTimeoutTests
                     "",
                     "");
                 await ReceiveResponse(connection, server.Context);
-
-                await Task.Delay(LongDelay);
-
-                await Assert.ThrowsAsync<IOException>(async () =>
-                {
-                    await connection.Send(
-                        "GET / HTTP/1.1",
-                        "",
-                        "");
-                    await ReceiveResponse(connection, server.Context);
-                });
+                await connection.WaitForConnectionClose().TimeoutAfter(LongDelay);
```

Comment:
Why were we calling Send before the last ReceiveResponse before. Is that needed to observe the connection close faster?